plugin/src/main/java/io/jenkins/plugins/remotingkafka/GlobalKafkaConfiguration.java

```diff
@@ -0,0 +1,43 @@
+package io.jenkins.plugins.remotingkafka;
+
+import hudson.Extension;
+import hudson.util.FormValidation;
+import io.jenkins.plugins.remotingkafka.kafkaclient.KafkaConsumerClient;
+import io.jenkins.plugins.remotingkafka.kafkaclient.KafkaProducerClient;
+import jenkins.model.GlobalConfiguration;
+import org.apache.commons.lang.StringUtils;
+import org.kohsuke.stapler.DataBoundSetter;
+import org.kohsuke.stapler.QueryParameter;
+
+@Extension
+public class GlobalKafkaConfiguration extends GlobalConfiguration {
+    private String connectionURL;
+
+    public GlobalKafkaConfiguration() {
+        load();
+    }
+
+    public static GlobalKafkaConfiguration get() {
+        return GlobalConfiguration.all().get(GlobalKafkaConfiguration.class);
+    }
+
+    public FormValidation doCheckConnectionURL(@QueryParameter String connectionURL) {
+        if (StringUtils.isEmpty(connectionURL)) {
```

Comment:
isBlank() would be better, because it would be able to catch strings like "    " in inputs